db.json

```diff
@@ -1647,7 +1698,7 @@
   },
   "application/sparql-query": {
     "source": "iana",
-    "extensions": ["rq"]
+    "extensions": ["rq","qs"]
   },
```

Comment:
.qs not mentioned on https://www.iana.org/assignments/media-types/application/sparql-query

Reply:
Added in https://gitlab.freedesktop.org/xdg/shared-mime-info/-/merge_requests/176 ; I cant see any evidence for .qs being used for this, and [github results](https://github.com/search?q=path%3A*.qs&type=code) are drowned out with it being used for https://doc.qt.io/qt-5/qtscript-index.html and other scripting languages.